IntelliTect.TestTools.TestFramework/IntelliTect.TestTools.TestFramework/TestBuilder.cs

```diff
@@ -86,6 +86,7 @@ public TestBuilder AddDependencyService<T>()
         /// <returns>This</returns>
         public TestBuilder AddDependencyInstance(object objToAdd)
         {
+            if (objToAdd == null) throw new ArgumentNullException(nameof(objToAdd));
```

Comment:
nit:
```suggestion
            if (objToAdd is null) throw new ArgumentNullException(nameof(objToAdd));
```

Reply:
Pretty sure you catch this every time I forget to do "is". :) Thanks!